input/input.css

```diff
@@ -0,0 +1,131 @@
+@import url(/style.css);
+
+.input-intro {
+  font-size: 1.25rem;
+  padding: 1rem 1rem;
+}
+.form-heading {
+  margin: 5px 0;
+  color: var(--darker-grey);
+}
+
+.form-login {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  width: 16rem;
+  height: 20rem;
+  border-radius: 10px;
```

Comment:
try to maintain only one unit through out class.. here for border-radius you can provide in rems

Reply:
It is a small px value so I did not converted it to rem.